scripts/validate_jsonschema.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Validates the JSON Schemas are valid.
+# Must be run from the repo's root folder.
+
+# Install JSON schema schema
+mkdir tmp
```

Comment:
Possibly `rm -rf tmp` so that it's fine to run this more than once (or run this after running something else that creates tmp.)

Reply:
I've added `-p` so there's no warning that the folder exists.

It is already fine running this more than once, the curl command replaces whatever may be there.